exercises/practice/space-age/.approaches/introduction.md

```diff
@@ -0,0 +1,111 @@
+# Introduction
+
+Suppose you are on Mercury.
+The [orbital period][wikipedia-orbital-period] of Mercury is approximately one fourth of an Earth year.
+This means that in one Earth year, Mercury will complete about four whole orbits around the sun.
+As a consequence, your 'age' on Mercury is about four times your age on Earth.
+
+More generally, your 'age' on a certain planet is the number of seconds since you birth, divided by the number of seconds it takes the planet to orbit the sun once.
```

Comment:
> I put the generally there to make the context switch from «you are on Mercury» to «you are on any one planet» explicit. Do you find this unnecessary?

Ah, that makes sense! I _was_ thinking it was unnecessary at the time, but after re-reading the text I think having the _generally_ does help with the context switch.

Reply:
There's also _In general_ if you prefer.

Reply:
I think the original _More generally_ flows better than _In general_ here because the text is going from the more specific (age on Mercury) to the more general case (age on any planet) - something that the _More generally_ phrasing highlight better than _In general_. I've just updated the suggested change.